src/logion/controllers/adapters/locrequestadapter.ts

```diff
@@ -133,21 +133,6 @@ export class LocRequestAdapter {
                 ...description
             };
         }
-        if (description.requesterAddress) {
-            const identityLoc = (await this.locRequestRepository.findBy({
-                expectedLocTypes: [ "Identity" ],
-                expectedIdentityLocType: "Polkadot",
-                expectedRequesterAddress: description.requesterAddress.address,
-                expectedOwnerAddress: description.ownerAddress,
-                expectedStatuses: [ "CLOSED" ]
-            })).find(loc => loc.getVoidInfo() === null);
-            if (identityLoc) {
-                return {
-                    identityLocId: identityLoc.id,
-                    ...identityLoc.getDescription()
-                }
-            }
-        }
         if (description.requesterIdentityLoc) {
```

Comment:
Are there cases where this is not set? Otherwise, remove the condition.

Reply:
No that was just meant to avoid using `description.requesterIdentityLoc!`, but I'll change this.